docs/guide/cli/analyze.md

```diff
@@ -0,0 +1,240 @@
+# analyze
+
+The `analyze` command is the core feature of the e18e CLI. It performs a comprehensive analysis of your JavaScript/TypeScript project to identify performance issues, potential optimizations, and areas for improvement.
+
+## Overview
+
+The analyze command runs multiple analysis plugins on your project:
+
+1. **ATTW (Are The Types Wrong)** - Validates TypeScript declaration files
+2. **Publint** - Checks package.json and publishing configuration
+3. **Module Replacements** - Identifies packages that can be replaced with faster alternatives
+4. **Dependency Analysis** - Analyzes your dependency tree for optimization opportunities
+
+## Usage
+
+```sh
+e18e-cli analyze [path] [options]
+```
+
+### Arguments
+
+- `path` (optional) - Path to analyze. Can be:
+  - A directory (defaults to current directory)
+  - A tarball file (`.tgz`)
+
+### Options
+
+| Option | Type | Default | Description |
+|--------|------|---------|-------------|
+| `--pack` | `enum` | `auto` | Package manager to use for packing. Options: `auto`, `npm`, `yarn`, `pnpm`, `bun`, `none` |
+| `--log-level` | `enum` | `info` | Set the log level. Options: `debug`, `info`, `warn`, `error` |
+| `--manifest` | `string[]` | `[]` | Path(s) to custom manifest file(s) for module replacements analysis |
+
+## Examples
+
+### Basic Analysis
+
+```sh
+# Analyze current directory
+e18e-cli analyze
+
+# Analyze specific directory
+e18e-cli analyze ./my-project
+
+# Analyze a tarball
+e18e-cli analyze ./my-package.tgz
+```
+
+## Analysis Types
```

Comment:
lets just drop this section since we already summarise them at the top and can link to the repos if needed in future.especially since the example output is very likely to change while we're still building